recipe_scrapers/panelinha.py

```diff
@@ -13,23 +13,22 @@ def host(cls):
         return "panelinha.com.br"
 
     def title(self):
-        return normalize_string(self.soup.find("h1").get_text())
```

Comment:
I think we can use `schema.org` here now:

```suggestion
    def title(self):
        return self.schema.title()
```

(or remove the method override from this scraper)